website/static/static/index.css

```diff
@@ -74,10 +85,21 @@ b {
     margin: 50px 0 0;
 }
 
+.maximized .demo {
+    flex-direction: column;
```

Comment:
Do we need `-ms-flex-direction` here as well.  We may want to consider adopting a CSS preprocessor to help with this in the future.